README.md

```diff
@@ -30,22 +28,50 @@ Use the `-port` option to specify the port number for the server to listen reque
 ./code-runner-server -port <port>
 ```
 
-## Send code files and arguments to server
-Use curl command to connect to the server and send request.
-Use multipart/form data option(-F) to provide the assignment submission file and the command-line arguments that are to be used when executing the code.
+## Code Runner as a docker image
+- A docker image of code runner is used as a base image for different language specific images.
+- Code Runner is not directly run, instead use the desired language specific image to run the assignments.
+- Prerequisite for this is that docker engine should be installed.
+- See [instructions](https://docs.docker.com/engine/installation/) for installing docker engine on different supported platforms.
+
+Use the following docker command to run the language specific image.
```

Comment:
"language specific image" is not accurate as it is actually the image for the required assignment environment.

Reply:
Agreed. Changed it.